examples/r-future/functions.R

```diff
@@ -0,0 +1,87 @@
+library(rsample)
+library(xgboost)
+library(Metrics)
+
+
+download_data <- function() {
+    if (!file.exists("births_data.rds")) {
+        download.file(
+            "https://saturn-public-data.s3.us-east-2.amazonaws.com/birth-data/births_2005.rds",
+            "births_data.rds"
+        )
+    }
+    births_raw_data <- readRDS("births_data.rds")
+
+    return(births_raw_data)
```

Comment:
R style guide says to not use the unnecessary return (this applies to I think all of your functions here)

https://style.tidyverse.org/functions.html?q=return#return

```suggestion
    readRDS("births_data.rds")
```

Reply:
Also, in general it's better to use `read_rds` and the equivalent functions from the `readr` package when you can. can we add a `library(readr)` and switch the function?